Sofa/Component/Topology/Mapping/src/sofa/component/topology/mapping/Edge2QuadTopologicalMapping.cpp

```diff
@@ -111,40 +110,39 @@ void Edge2QuadTopologicalMapping::init()
         {
             msg_info() << "Edge2QuadTopologicalMapping - to = quad";
 
-            container::dynamic::QuadSetTopologyModifier *to_tstm;
-            toModel->getContext()->get(to_tstm);
-
-            container::dynamic::QuadSetTopologyContainer *to_tstc;
-            toModel->getContext()->get(to_tstc);
+            container::dynamic::QuadSetTopologyModifier *to_modifier;
+            toModel->getContext()->get(to_modifier);
 
-            const sofa::type::vector<Edge> &edgeArray=fromModel->getEdges();
-
-            sofa::type::vector<Index>& Loc2GlobVec = *(Loc2GlobDataVec.beginEdit());
+            container::dynamic::QuadSetTopologyContainer *to_container;
+            toModel->getContext()->get(to_container);
 
+            const sofa::type::vector<Edge>& edgeArray = fromModel->getEdges();
+            
+            auto Loc2GlobVec = sofa::helper::getWriteOnlyAccessor(Loc2GlobDataVec);
             Loc2GlobVec.clear();
             In2OutMap.clear();
 
             // CREATION of the points (new DOFs for the output topology) along the circles around each point of the input topology
 
-            Vec X0(1.,0.,0.);
-            Vec Y0;
-            Vec Z0;
+            Vec3 X0(1.,0.,0.);
+            Vec3 Y0;
+            Vec3 Z0;
 
             if (ellipse){
                 Z0 = d_focalAxis.getValue();
                 Z0.normalize();
                 Y0 = cross(Z0,X0);
             } else {
-                Y0[0] = (Real) (0.0); Y0[1] = (Real) (1.0); Y0[2] = (Real) (0.0);
-                Z0[0] = (Real) (0.0); Z0[1] = (Real) (0.0); Z0[2] = (Real) (1.0);
+                Y0[0] = (SReal) (0.0); Y0[1] = (SReal) (1.0); Y0[2] = (SReal) (0.0);
+                Z0[0] = (SReal) (0.0); Z0[1] = (SReal) (0.0); Z0[2] = (SReal) (1.0);
```

Comment:
You could use 0_sreal and 1_sreal. You could put all instructions in different lines

Reply:
I keep this line Asis for the moment as I just did some quick replace. 
I'm working on other PR to refactor this code.